openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/storages/TrafficGraphStorage.java

```diff
@@ -0,0 +1,618 @@
+/*  This file is part of Openrouteservice.
+ *
+ *  Openrouteservice is free software; you can redistribute it and/or modify it under the terms of the
+ *  GNU Lesser General Public License as published by the Free Software Foundation; either version 2.1
+ *  of the License, or (at your option) any later version.
+
+ *  This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *  See the GNU Lesser General Public License for more details.
+
+ *  You should have received a copy of the GNU Lesser General Public License along with this library;
+ *  if not, see <https://www.gnu.org/licenses/>.
+ */
+package org.heigit.ors.routing.graphhopper.extensions.storages;
+
+import com.graphhopper.storage.DataAccess;
+import com.graphhopper.storage.Directory;
+import com.graphhopper.storage.Graph;
+import com.graphhopper.storage.GraphExtension;
+import com.graphhopper.storage.RAMDirectory;
+import org.heigit.ors.routing.graphhopper.extensions.reader.traffic.TrafficEnums;
+
+import java.util.Calendar;
+import java.util.TimeZone;
+
+/**
+ * Graph storage class for the Border Restriction routing
+ */
+public class TrafficGraphStorage implements GraphExtension {
+
```

Comment:
Why not use enums here?

Reply:
@MichaelsJP

Reply:
@HendrikLeuschner  @sfendrich I moved TrafficRelevantWayTypes and RoadTypes to enums  and refactored their usages